prowler/providers/aws/services/elasticache/elasticache_service.py

```diff
@@ -132,7 +151,7 @@ class Cluster(BaseModel):
     id: str
     arn: str
     region: str
-    security_groups: list[str] = []
+    engine: str
```

Comment:
Why this change?

Reply:
Because the security_groups attribute is not needed, and I need the engine to know if the cluster is Redis or Memcached.

Reply:
Good to know, thanks!